src/lib/libraries/decks/translate-video.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * @fileoverview
+ * Utility function to look up whether a translated video exists for a locale.
+ */
+
+// TODO: ja-Hira are copies of ja, replace with real ja-Hira when available.
```

Comment:
weren't you telling me there were some relationships, like brasilian portuguese -> portuguese, that had explicit failover/backup relationships? Was that in scratch-l10n?

Could it make sense to use something like that here?
```
const failovers = {
      'ja-Hira': 'ja'
}
const findWithFailover(object, locale, failovers) {
    if (object.hasOwnProperty(locale)) return object[locale];
    const failover = failovers[locale];
    if (failover) return findWithFailover(object, failover, failovers);
    if (object.hasOwnProperty('en')) return object['en'];
    return null;
}
```
then...
```
const translateVideo = (videoId, locale) => {
    // if the videoId is not recognized in the videos object, assume it's already a video id on wistia
    return findWithFailover(videos[videoId], locale, failovers);
};
```

Reply:
We don't have any fail-overs. I was mentioning it might be nice in the string translations to be able to have the fallback language be something other than English, but it's not supported by react-intl. And it could be problematic - failing over between the two Chinese systems would probably not be acceptable.
Using the ja video for ja-Hira is only temporary. I checked with Yumiko, and we just need to redo the video - her voice over can be used for both. If people start asking for some other language video instead of English, then it might be worth looking into adding something like this. Right now I think it over complicates the videos for a situation that is the exception not the rule.